src/Common/PlanetParameters/EarthConstants.jl

```diff
@@ -0,0 +1,83 @@
+function PlanetConstants{FT}(::Type{EarthParameters};
+                             k_Karman =          FT(0.4),
```

Comment:
Can we please retain the long-form definition and information on units we had in PlanetParameters, at least as a comment? This is important for readability.

Reply:
I mean: have the long-form definitions stated next to the numerical values, rather than in a separate file?

Reply:
But it's also ok to leave it if it is difficult to incorporate the constant names with the numerical values.

Reply:
The documentation for the parameters are in the `PlanearytConstants.jl`. I think it'd be easier to keep the docs there and focus on values in the planet files, but I can add them to both if you'd like.

Reply:
Right, I saw that. What I mean is that it would be nice to keep documentation near where the numerical values are defined (e.g., to see units next to values).

Reply:
I'll add them back in, I think it'll be fine either way.